build.groovy

```diff
@@ -63,29 +58,53 @@ node("${env.OS.toLowerCase()}") {
     }
 
     stage("Create environment") {
+        
         env.STAGE_STATUS = "Creating conda environment"
-        sh '''
+        
+        if (env.OS.toLowerCase() == "mac") {
+          condaPath = "/tmp/" + sh(script: '{ date "+%H:%M:%S:%m"; echo $WORKSPACE; } | md5 | tr -d "\n";', returnStdout: true) 
+
+          sh """
+            curl -o miniconda.sh  https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+            bash miniconda.sh -b -p ${condaPath}
+            """
+        } else {
+          condaPath = "/home/jenkins/.conda/"
+        } 
+        
+        println(condaPath)
+        isisEnv.add("PATH=${pwd()}/install/bin:$condaPath/envs/isis/bin:$condaPath/bin:${env.PATH}")
+         
+        sh """
             # Use the conda cache running on the Jenkins host
             # conda config --set channel_alias http://dmz-jenkins.wr.usgs.gov
+            export PATH="${condaPath}/bin:${env.PATH}"
+            echo $PATH
+            which conda
             conda config --set always_yes True
             conda config --set ssl_verify false 
             conda create -n isis python=3
-        '''
-
+        """
+          
+        
         if (env.OS.toLowerCase() == "centos") {
             sh 'conda env update -n isis -f environment_gcc4.yml --prune'
         } else {
-            sh 'conda env update -n isis -f environment.yml --prune'
+          sh """
+            export PATH="${condaPath}/bin:${env.PATH}"
+            which conda
```

Comment:
This looks like it's duplicated from above.

Reply:
I'll see if `withEnv` works within a stage to remove the duplicate exports.